port/unix/omrsysinfo.c

```diff
@@ -1154,6 +1154,10 @@ omrsysinfo_map_ppc_processor(const char *processorName)
 		rc = OMR_PROCESSOR_PPC_P10;
 	} else if (0 == strncasecmp(processorName, "power11", 7)) {
 		rc = OMR_PROCESSOR_PPC_P11;
+	} else if (0 == strncasecmp(processorName, "power1", 6)) {
+		// Punt the long-future anticipated cases to the latest we know
```

Comment:
minor nit, port library doesn't use `//` for comments ... i have also reworded the comment to improve readability
```suggestion
		/* Defer the long-term anticipated cases to the latest known state. */
```